shared/login/register/set-public-name/container.js

```diff
@@ -38,9 +38,23 @@ class _SetPublicName extends Component<void, ContainerProps, State> {
 
     this.state = {
       deviceName: null,
+      deviceNameError: null,
```

Comment:
why isn't the prop just being updated in the store, why move this to local state?